### ISSUE TITLE
KeyError's in log

### ISSUE THREAD

**Issue by jjjordanmsft** [Microsoft/ApplicationInsights-Docker#22]
Cannot determine severity here, but I have yet to see any data on the portal (after ~24 hours).  This is running under Docker 1.11.1 on Ubuntu LTS 14.04, I see these messages (repeated 3-4 times) in the container's stderr logs:

```
Traceback (most recent call last):
  File "python/bootstrap.py", line 49, in <module>
    methods[method]()
  File "python/bootstrap.py", line 44, in <lambda>
    'inject': lambda: program.run_injector(docker_socket=_docker_socket, docker_info_path=_docker_info_path),
  File "/usr/docker/ApplicationInsights-Docker/build/docker/python/appinsights/program.py", line 36, in run_injector
    injector.start()
  File "/usr/docker/ApplicationInsights-Docker/build/docker/python/appinsights/dockerinjector.py", line 78, in start
    self._docker_wrapper.get_events()))
  File "/usr/lib/python3.4/concurrent/futures/_base.py", line 541, in map
    fs = [self.submit(fn, *args) for args in zip(*iterables)]
  File "/usr/lib/python3.4/concurrent/futures/_base.py", line 541, in <listcomp>
    fs = [self.submit(fn, *args) for args in zip(*iterables)]
  File "/usr/docker/ApplicationInsights-Docker/build/docker/python/appinsights/dockerwrapper.py", line 72, in get_events
    event['Id'] = event['id']
KeyError: 'id'
```

**Comment by Yonisha** [comment]
Hi jjjordanmsft,
Docker version 1.11.1 released two weeks ago and includes some breaking changes.
Therefore, this version is not supported and currently no ETA for supporting it.

**Comment by Yonisha** [comment]
Hi jjjordanmsft,

The issue has been fixed in v0.9.2.
Can you please pull the latest version and check if the issue resolved?

**Comment by jjjordanmsft** [comment]
This appears to be fixed now, thanks!
